setup_kubernetes.sh

```diff
@@ -410,7 +453,18 @@ create_vfs(){
       sed -i "/$switchdev_scripts_name/d" /etc/rc.d/rc.local
    fi
 
-   echo "$switchdev_path/$switchdev_scripts_name -i $interface -v $vfs_num" >> /etc/rc.d/rc.local
+   if [[ "$is_bond" == "false" ]]
+   then
+      echo "$switchdev_path/$switchdev_scripts_name -i $interface -v $vfs_num" >> /etc/rc.d/rc.local
+      bash "$switchdev_path"/"$switchdev_scripts_name" -i "$interface" -v "$vfs_num"
+   elif [[ "$is_bond" == "true" ]]
+   then
+      echo "$switchdev_path/$switchdev_scripts_name -i $slave1 -v $vfs_num" >> /etc/rc.d/rc.local
+      echo "$switchdev_path/$switchdev_scripts_name -i $slave2 -v $vfs_num" >> /etc/rc.d/rc.local
+      bash "$switchdev_path"/"$switchdev_scripts_name" -i "$slave1" -v "$vfs_num"
+      bash "$switchdev_path"/"$switchdev_scripts_name" -i "$slave2" -v "$vfs_num"
+   fi
```

Comment:
did you test bond with reboot ?

Reply:
tested it on a master node, it persisted and the vf lag was successfully configured. cant test it on worker node because there is no setup to test it on (i am not using oz setup). though i will need to a add another patchset to fix a minor issue.